i18n/bundle/bundle.go

```diff
@@ -129,9 +129,30 @@ func (b *Bundle) AddTranslation(lang *language.Language, translations ...transla
 	}
 }
 
-// Translations returns all translations in the bundle.
-func (b *Bundle) Translations() map[string]map[string]translation.Translation {
-	return b.translations
+// LanguageTags returns the tags of all languages that that have been added.
+func (b *Bundle) LanguageTags() []string {
+	tags := make([]string, len(b.translations))
+	i := 0
+	for k := range b.translations {
+		tags[i] = k
```

Comment:
same here – consider using `append()` and maybe using a named return value to simplify things

```go
func (b *Bundle) LanguageTags() (tags []string) {
    for _, translation := range b.translations {
        tags = append(tags, translation)
    }
    return
}
```